l10n/de_DE/LC_MESSAGES/remoteevent.po

```diff
@@ -1606,6 +1747,110 @@ msgstr "Teilnehmerrolle"
 msgid "(not counted)"
 msgstr "(nicht gezählt)"
 
+#~ msgid "[FULL] "
```

Comment:
Delete unused translations?

Reply:
I'd keep them, in case there should be translations in the future. Otherwise, one would have to update with the catalogue file first.